### ISSUE TITLE
npm: cross-spawn-async is said to be deprecated

### ISSUE THREAD

**Issue by lthms** [GreenPix/dilia#22]
``` bash
[lethom@ideepad dilia]$ npm install
npm WARN deprecated cross-spawn-async@2.2.4: cross-spawn no longer requires a build toolchain, use it instead!
```

**Comment by Nemikolh** [comment]
This lib is used by `node-sass`, I just noticed that we could upgrade it to `3.8.0`. Might fix the issue.

**Comment by Nemikolh** [comment]
Fixed in eb2af4e.
